CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Gulp Sass
+
+Gulp Sass is a very light-weight [Gulp](https://github.com/gulpjs/gulp) wrapper for [`node-sass`](https://github.com/sass/node-sass), which in turn is a Node binding for [`libsass`](https://github.com/sass/libsass), which in turn is a port of [`Sass`](https://github.com/sass/sass).
+
+## Submitting Issues
+
+* Before creating a new issue, perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=repo%3Adlmanning%2Fgulp-sass+repo%3Asass%2Fnode-sass+repo%3Asass%2Flibsass+repo%3Asass%2Fsass+repo%3Asass-eyeglass%2Feyeglass) in the Gulp Sass, Node Sass, Libsass, and main Sass repos to see if a similar issue has already been submitted.
+* You can create an issue [here](https://github.com/dlmanning/gulp-sass/issues). Please include as many details as possible in your report.
+* Issue titles should be descriptive, explaining at the high level what it is about.
+* Please include the version of `gulp-sass` you are using.
+* Do not open a [pull request](#pull-requests) to resolve an issue without first receiving feedback from a `collaborator` or `owner` and having them agree on a solution forward.
+* Include screenshots and animated GIFs whenever possible; they are immensely helpful.
+* Issues that have a number of sub-items that need to be complete should use [task lists](https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments) to track the sub-items in the main issue comment.
+
+
+## Pull Requests
+
+* **DO NOT ISSUE A PULL REQUEST WITHOUT FIRST [SUBMITTING AN ISSUE](#submitting-issues)**
+* Pull requests should reference their related issues. If the pull request closes an issue, [please reference its closing in your commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/). Pull requests not referencing any issues will be closed.
+* Pull request titles should be descriptive, explaining at the high level what it is doing, and should be written in the same style as [Git commit messages](#git-commit-messages).
+* Update the `CHANGELOG` with the changes made by your pull request, making sure to use the proper [Emoji](#emoji-cheatsheet).
+* Follow our JavaScript styleguides. Tests will fail if you do not.
+* Ensure that you have [EditorConfig](http://editorconfig.org/) installed in your editor of choice and that it is functioning properly.
+* Do not squash or rebase your commits when submitting a Pull Request. It makes it much harder to follow your work and make incremental changes.
+* Update the [CHANGELOG](#maintaining-thechangelog) with your changes.
+* Ensure no Emoji tags are used in the title of your Pull Request
+
+### Git Commit Messages
+
+* Use the present tense (`"Add feature"` not `"Added Feature"`)
+* Use the imperative mood (`"Move cursor to…"` not `"Moves cursor to…"`)
+* Limit the first line to 72 characters or less
+* Include relevant Emoji from our [Emoji cheatsheet](#emoji-cheatsheet)
```

Comment:
I always found this a bit silly on javascript repos, most of the time I'm looking at the log in my terminal, not on github